src/algorithms/expected_consensus.md

```diff
@@ -161,115 +86,43 @@ We use the ECVRF algorithm from [Goldberg et al. Section 5](https://tools.ietf.o
 
 ### Running a leader election
 
-Now, a miner must also check whether they are eligible to mine a block in this round. For how Election Proofs are validated, see [election validation](mining.md#election-validation).
+Now, a miner must also check whether they are eligible to mine a block in this round.
 
 To do so, the miner will use tickets from K rounds back as randomness to uniformly draw a value from 0 to 1. Comparing this value to their power, they determine whether they are eligible to mine. A user's `power` is defined as the ratio of the amount of storage they proved as of their last PoSt submission to the total storage in the network as of the current block.
 
 We use the ECVRF algorithm (must yield a pseudorandom, deterministic output) from [Goldberg et al. Section 5](https://tools.ietf.org/html/draft-irtf-cfrg-vrf-04#page-10), with:
   - Sha256 for our hashing function
   - Secp256k1 for our curve
 
-If the miner scratches a winning ticket in this round, it can use newEP, along with a newTicket to generate and publish a new block (see [Block Generation](#block-generation)). Otherwise, it waits to hear of another block generated in this round.
+If the miner scratches a winning ticket in this round, it can use newEP, along with a newTicket to generate and publish a new block. Otherwise, it waits to hear of another block generated in this round.
 
-It is important to note that every block contains two artifacts: one, a ticket derived from last block's ticket to prove that they have waited the appropriate delay, and two, an election proof derived from the ticket `K` rounds back used to run leader election.
+It is important to note that every block contains two artifacts: one, a ticket derived from last block's ticket to extend the ticket-chain, and two, an election proof derived from the ticket `K` rounds back used to run leader election.
 
-### Election Validation
+Succinctly, the process of crafting a new ElectionProof in round N is as follows. We use:
 
-For election proof generation, see [checking election results](expected-consensus.md#checking-election-results).
+    The ECVRF algorithm (must yield a pseudorandom, deterministic output) from Goldberg et al. Section 5, with:
+        Sha256 for our hashing function
+        Secp256k1 for our curve
 
-In order to determine that the mined block was generated by an eligible miner, one must check its `ElectionProof`.
+Note: We draw the miner power from the prior round. This means that if a miner wins a block on their ProvingPeriodEnd even if they have not yet resubmitted a PoSt, they retain their power (until the next round).
```

Comment:
This is talking specifically about the edge case that is: what if I win a block in the round in which i am challenged (and don't submit). Do I still win or is it forfeited. So this is just meant to avoid fencepost errors. I'll clarify